src/extension/features/reports/income-expense-hover-highlight/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'IncomeVsExpenseHoverHighlight',
+  type: 'checkbox',
+  default: true,
+  section: 'reports',
+  title: 'Highlight Income vs Expense Row on Hover',
+  description: 'Provides a highlight over the currently hovered row on the native YNAB Income vs Expense report.'
```

Comment:
Glad you included the word 'native' here to indicate this is related to the native report and not the Toolkit reports.